roles/keystone-setup/tasks/main.yml

```diff
@@ -27,7 +52,25 @@
                     internal_url={{ item.internal_url }}
                     admin_url={{ item.admin_url }}
                     region=RegionOne
-                    token={{ secrets.admin_token }}
+                    token={{ admin_token }}
   # TODO refactor keystone.services data to be easier to pull out individual services
   with_items: keystone.services
   when: endpoints[item.name] is defined and item.name == 'keystone'
+
+- name: remote admin_token from keystone api
+  lineinfile: dest=/etc/keystone/keystone.conf
+              line="admin_token = {{ admin_token }}"
```

Comment:
Sure I could do that, or maybe what I should do is just re-template that file. That seems better and follows what I was doing for keystone-paste.